### ISSUE TITLE
train loss on cifar10 dataset is nan

### ISSUE THREAD

**Issue by diadestiny** [VinAIResearch/WaveDiff#7]
Thanks for your work!
Why is the loss training on cifar10 dataset nan? I just download the code and run the script (bash run.sh cifar10 train 1)
![image](https://github.com/VinAIResearch/WaveDiff/assets/44188454/4a0d4b77-19a2-4151-ae96-3a2897620e32)

**Comment by hao-pt** [comment]
hi there, after cloning this repo for quick check, I experience no nan loss as in your case. However, this issue might be due to the environment setting. I strongly recommend you use the same version of PyTorch 1.10.0 as ours to avoid any unexpected cases.
<img width="1503" alt="image" src="https://github.com/VinAIResearch/WaveDiff/assets/31931630/67bc78f7-2b38-47e5-8058-ea5dc2314137">

**Comment by Amarkr1** [comment]
Indeed, I reverified this. A different PyTorch version other than 1.10.0 results in nan. Not sure what leads to this.
